src/styles/main.css

```diff
@@ -241,6 +249,37 @@
 
   margin-right: 0.3rem;
 }
+.form-error { /* validation message for the field */
+    @apply text-red text-caption uppercase tracking-wide block mt-1;
+}
+.form-error-on-dark-bg {
+    @apply form-error text-lightred;
+}
+
+.lang-menu-open .lang-flag {
+  @apply mr-2;
+}
+
+.min-h-stretch {
+  /* Fill all available vertical space. It works predictable for mobile,
+  because it doesn't take the address bar into account. */
+  @apply min-h-screen;
+  min-height: stretch;
+}
```

Comment:
I see that you've added new classes for this footer behaviour. We already have some defined that we use in the existing application footer. can't we just use what already exists? maybe there is something different that I'm not aware of

Reply:
I haven't found anything similar to [this behavior](https://codepen.io/chriscoyier/pen/RRbKrL) in existing styles. Maybe I missed something?

These styles are just a little abstraction above several flex-related styles. I decided to create them because it was more convenient to use meaningful class names in a mess of HTML elements among all other flex-related classes.

Reply:
We do, our footer uses `flex-grow` to achieve this: https://github.com/cambiatus/frontend/blob/b62fdedc696f8bc597c453296ca1c98c1bf872c6/src/elm/Session/LoggedIn.elm#L288